games-strategy/freeciv/freeciv-2.5.10.recipe

```diff
@@ -0,0 +1,119 @@
+over a network, and an AI which gives most people a run for their money.
+Freeciv aims to be mostly rule-compatible with Civilization II [tm], \
+published by Sid Meier and Microprose [tm]. A few rules are different \
+where we think it makes more sense, and we have lots and lots of \
+adjustable parameters to make customizing games possible.
+Freeciv has been implemented completely independently of Civilization; \
+you do not need to own Civilization to play Freeciv."
+HOMEPAGE="http://www.freeciv.org"
+COPYRIGHT="1996-2017 The Freeciv Team"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="http://downloads.sourceforge.net/project/freeciv/Freeciv%202.5/$portVersion/freeciv-$portVersion.tar.bz2"
+CHECKSUM_SHA256="c8a14156562dce2f0a7b96390c2d5a35314f1124c448782a6a47b0c51cfcb302"
+ADDITIONAL_FILES="freeciv.rdef.in"
+PATCHES="freeciv-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+GLOBAL_WRITABLE_FILES="
+	settings/freeciv/database.lua keep-old
+	"
+
+PROVIDES="
+	freeciv$secondaryArchSuffix = $portVersion
+	app:freeciv$secondaryArchSuffix = $portVersion
+	cmd:freeciv_manual$secondaryArchSuffix
+	cmd:freeciv_sdl$secondaryArchSuffix
+	cmd:freeciv_server$secondaryArchSuffix
+	lib:libfreeciv$secondaryArchSuffix
+	lib:libfreeciv_srv$secondaryArchSuffix
+	"
+
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libbz2$secondaryArchSuffix
+	lib:libcurl$secondaryArchSuffix
+	lib:libcrypto$secondaryArchSuffix
+	lib:libfreetype$secondaryArchSuffix
+	lib:libGL$secondaryArchSuffix
+	lib:libiconv$secondaryArchSuffix
+	lib:libintl$secondaryArchSuffix
+	lib:liblua$secondaryArchSuffix
+	lib:liblzma$secondaryArchSuffix
+	lib:libreadline$secondaryArchSuffix
+	lib:libsdl_1.2$secondaryArchSuffix
+	lib:libsdl_gfx$secondaryArchSuffix
+	lib:libsdl_image_1.2$secondaryArchSuffix
+	lib:libsdl_mixer_1.2$secondaryArchSuffix
+	lib:libsdl_ttf_2.0$secondaryArchSuffix
+	lib:libssl$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libbz2$secondaryArchSuffix
+	devel:libcurl$secondaryArchSuffix
+	devel:libfreetype$secondaryArchSuffix
+	devel:libiconv$secondaryArchSuffix
+	devel:libintl$secondaryArchSuffix
+	devel:liblua$secondaryArchSuffix >= 5.2
+	devel:liblzma$secondaryArchSuffix
+	devel:libreadline$secondaryArchSuffix
+	devel:libsdl$secondaryArchSuffix
+	devel:libsdl_gfx$secondaryArchSuffix
+	devel:libsdl_image$secondaryArchSuffix
+	devel:libsdl_mixer$secondaryArchSuffix
+	devel:libsdl_ttf$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:gcc$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:make
+	cmd:autoconf
+	cmd:aclocal
+	cmd:libtool
+	cmd:gettext$secondaryArchSuffix
+	cmd:find
+	cmd:pkg_config$secondaryArchSuffix
+	"
+
+BUILD()
+{
+	autoreconf -vfi
+	runConfigure ./configure --datarootdir=$dataRootDir --datadir=$dataDir \
+		--enable-client=sdl --enable-fcmp=no --enable-sys-lua --enable-shared #--enable-mapimg=no
```

Comment:
```diff
 runConfigure ./configure --datarootdir=$dataRootDir --datadir=$dataDir \
-		--enable-client=sdl --enable-fcmp=no --enable-sys-lua --enable-shared #--enable-mapimg=no
+		--enable-client=sdl --enable-fcmp=no --enable-sys-lua \
+		--enable-shared #--enable-mapimg=no
```
BTW, you might wish to use `runConfigure  --omit-dirs binDir ./configure --bindir=$appsDir` ... to be able to drop the `mv $binDir/freeciv-sdl $appsDir/Freeciv`. Oops, you will still need it, to drop to `-sdl` suffix. So this is not a good idea, after all, unless you choose to keep the suffix.

Reply:
This has been a discussion in the past, or we keep all FreeCiv stuff in one directory or we use the system directories ... ps, not all related to FreeCiv ...